mlflow/utils/autologging_utils/safety.py

```diff
@@ -59,7 +59,9 @@ def safe_function(*args, **kwargs):
             if is_testing():
                 raise
             else:
-                _logger.warning("Encountered unexpected error during autologging: %s", e)
+                import traceback
+                _logger.warning("Encountered unexpected error during autologging:\n %s",
+                                traceback.format_exc(e))
```

Comment:
You might already know this but you can set `MLFLOW_AUTOLOGGING_TESTING` to `true` and avoid suppressing errors encountered during autologgigng.

Reply:
oh. nice !